src/utils/xccdf2inspec.ts

```diff
@@ -131,8 +131,8 @@ export function impactNumberToSeverityString(impact: number): string {
   }
 }
 
-export function inspecControlToRubyCode(control: InSpecControl, lineLength?: number): string {
-  let result = '# encoding: UTF-8\n\n'
+export function inspecControlToRubyCode(control: InSpecControl, lineLength?: number, showUtfEncoding?: boolean): string {
```

Comment:
Encoding header for xccdf and spreadsheet treat the expected output of setting the encodingHeader variable in completely opposite ways at least according to the description for the flags; however, in both instances those variables are being passed straight to this function.

Please either fix calls to this function so that the behavior as described in the descriptions occurs or change the description for spreadsheet to make it so both are 'add the header'.